src/components/Layout/DrawerNavigation.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { FunctionComponent, useCallback, useContext } from "react";
+import {
+  Linking,
+  StyleSheet,
+  Alert,
+  TouchableOpacity,
+  View
+} from "react-native";
+import { SafeAreaView } from "react-navigation";
+import {
+  DrawerContentComponentProps,
+  DrawerActions
+} from "react-navigation-drawer";
+import { useLogout } from "../../hooks/useLogout";
+import { AppText } from "./AppText";
+import { size, color, fontSize } from "../../common/styles";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import { HelpModalContext } from "../../context/help";
+import { useDrawerContext, DrawerButtons } from "../../context/drawer";
+import Constants from "expo-constants";
+import { APP_BUILD_VERSION } from "../../config";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    position: "relative"
+  },
+  navLink: {
+    flexDirection: "row",
+    alignItems: "center",
+    paddingHorizontal: size(4),
+    paddingVertical: size(2.5),
+    borderBottomColor: color("grey", 20),
+    borderBottomWidth: 1
+  },
+  navLinkText: {
+    color: color("blue", 50),
```

Comment:
For all the default AppText, there's no need for this since they are blue 50 by default